mysite/templates/base.html

```diff
@@ -1,21 +1,27 @@
 {% load static %}
+{% load bootstrap4 %}
```

Comment:
needed this import here as well because of the tags on L12 and L13 (surprised me because we already imported it in base.html)